java/ql/integration-tests/all-platforms/java/multi-release-jar-java11/test.py

```diff
@@ -1,15 +1,24 @@
-import sys
+import commands
 
-from create_database_utils import *
-import subprocess
-import os
 
-try_use_java11()
-
-os.mkdir("mod1obj")
-os.mkdir("mod2obj")
-
-subprocess.check_call(["javac", "mod1/module-info.java", "mod1/mod1pkg/Mod1Class.java", "-d", "mod1obj"])
-subprocess.check_call(["jar", "-c", "-f", "mod1.jar", "-C", "mod1obj", "mod1pkg/Mod1Class.class", "--release", "9", "-C", "mod1obj", "module-info.class"])
-
-run_codeql_database_create(["javac mod2/mod2pkg/User.java mod2/module-info.java -d mod2obj -p mod1.jar"], lang="java")
+def test(codeql, use_java_11, java):
+    commands.run(["javac", "mod1/module-info.java", "mod1/mod1pkg/Mod1Class.java", "-d", "mod1obj"])
+    commands.run(
+        [
+            "jar",
+            "-c",
+            "-f",
+            "mod1.jar",
+            "-C",
+            "mod1obj",
+            "mod1pkg/Mod1Class.class",
+            "--release",
+            "9",
+            "-C",
+            "mod1obj",
+            "module-info.class",
+        ]
+    )
```

Comment:
Stylistic nit, you can use strings instead of lists of strings, and combine the two calls:
```suggestion
    commands.run(
        "javac mod1/module-info.java mod1/mod1pkg/Mod1Class.java -d mod1obj",
        "jar -c -f mod1.jar -C mod1obj mod1pkg/Mod1Class.class --release 9 -C mod1obj module-info.class",
    )
```